### ISSUE TITLE
Ability to select multiple filters simultaneously in Django administration

### ISSUE THREAD

**Issue by rhb123** [BirkbeckCTP/janeway#3892]
**Is your feature request related to a problem? Please describe.**

I currently need to audit our typesetting bill to check we are being charged correctly for different article types. As such, to use the Django administration search page at https://www.openlibhums.org/admin/submission/article/?q=Review&stage__exact=Published I would need to be able to select multiple article stages simultaneously in the search (Typesetting and Proofing and Prepub and Published, as typeset articles may be in any of these stages) and search alongside a number of different Review sections simultaneously (each journal has a different selection for this and they are named slightly differently, so I need to check all of these) to check that typeset articles published in a set date range (Oct-Dec 23 in this case) have been properly billed by our typesetters.

Currently the Django administration page doesn't let you select multiple categories at once to narrow down the results, so these are unfortunately far too broad to work with at present, and only let me work with one of the selected filters at a time, rather than multiple, which means that it's unusable for my purposes at present.

**Describe the solution you'd like**
It would be great to get a version of the search that allows you to select multiple filter items at once, ie, multiple typesetting stages and multiple different sections to search across at once, which would massively speed up checking this info across 30 journals manually.

**Describe alternatives you've considered**
The alternative is going through all our journals manually to check these one by one, which I'm having to do and which is very time consuming, so it would be brilliant to get a fix for this so that these checks can be automated in future.

Thank you in advance for your help!

**Comment by ajrbyers** [comment]
- [ ] @ajrbyers move this to the APC plugin and confirm with Rose this will be okay

**Comment by S-Haime** [comment]
Is this covered by RedDirective, @StephDriver ?

**Comment by StephDriver** [comment]
> Is this covered by RedDirective, [@StephDriver](https://github.com/StephDriver) ?

Yes @S-Haime, good find. it is a duplicate or at least has been overtaken by that work.  Closing this.
